src/test/java/seedu/address/TestApp.java

```
@@ -62,6 +65,15 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
         return userPrefs;
     }
 
+    /**
+     * Returns a defensive copy of the address book content stored inside the storage file.
+     *
+     * @throws IOException if the storage file for the address book cannot be found.
+     * @throws DataConversionException if the file is not in the correct format.
+     */
+    public AddressBook readStorageAddressBook() throws DataConversionException, IOException {
```

on more thought, I'm wondering if there's a need to throw these exceptions since this class is `TestApp` which is a specially tailored class for testing purposes. As such, not sure whether we can actually just have a try-catch block here, and then we can just `throw new AssertionError()` or something, so that classes calling this method don't need to have the `throw Exception` in their method signature.